helix-common/src/main/java/org/apache/helix/SystemPropertyKeys.java

```diff
@@ -82,4 +82,6 @@
   // System Property Metadata Store Directory Server endpoint key
   public static final String MSDS_SERVER_ENDPOINT_KEY =
       MetadataStoreRoutingConstants.MSDS_SERVER_ENDPOINT_KEY;
+
+  public static final String STATEUPDATEUTIL_ERROR_LOG_ENABLED = "helix.StateUpdateUtil.errorLog.enabled";
```

Comment:
minor: "helix.StateUpdateUtil.errorLogToZK.enabled"?  We still log error to local logfile, right?

Reply:
This is not minor, this is a MAJOR, a very good point. Let me make sure we have error level log out.